src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs

```diff
@@ -22,7 +22,7 @@ public static class CommonUtilities
             {
                 using (var pdb = new MemoryStream())
                 {
-                    if (PlatformHelper.IsMono)
+                    if (PlatformHelper.IsMono || !PlatformHelper.IsWindows)
```

Comment:
Yeah, we shouldn't care about the pdbs. These assemblies are generated either for retrieving EF related information or to compile the razor templates. 
Should be safe to always pass pdbstream as null